src/xcode/ENA/ENA/Source/Services/ExposureSubmissionService.swift

```diff
@@ -179,11 +179,28 @@ class ENAExposureSubmissionService: ExposureSubmissionService {
 				return
 			}
 
-			guard let keys = keys, !keys.isEmpty else {
+			guard var keys = keys, !keys.isEmpty else {
```

Comment:
@thomasaugsten If keys are not ordered desc by creation time in the array the code would:
- cut the wrong keys
- assign the wrong transmissionRisk defaults.

To be on the safe side I recommend to order the diagnosisKeys by creationtimestam

Reply:
there are sorted by _rollingStartNumber.  There is no timestamp. I would prefer to not reorder the keys